node/src/BaseClient.ts

```diff
@@ -977,10 +979,15 @@ export class BaseClient {
      * ```
      */
     public async getex(
-        key: string,
-        options?: "persist" | { type: TimeUnit; duration: number },
+        key: GlideString,
+        options?: {
+            cmdOptions: "persist" | { type: TimeUnit; duration: number };
```

Comment:
```suggestion
            expiry: "persist" | { type: TimeUnit; duration: number };
```
Rename to `expiry` to align with `set` command options and update tests